wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -49,7 +49,7 @@ Refer to [this thread](https://osu.ppy.sh/forum/t/178700) for alternative diffic
 Overall
 --------
 
-Overall rules and guidelines apply to every kind of osu! difficulty. Rhythm related guidelines apply to approximately 180 BPM maps. If your song is drastically faster or slower, some variables might be different. Apply reasonable judgement in these cases.
+Overall rules and guidelines apply to every kind of osu! difficulty. Rhythm related guidelines apply to approximately 180 BPM maps with 4/4 time signatures. If your song is drastically faster or slower, some variables might be different. [This article](/wiki/Ranking_Criteria/osu!/Scaling_BPM) serves as a reference for dealing with differing BPMs more accurately and is as such this Ranking Criteria and applying to all difficulties.
```

Comment:
this section seems poorly worded, do you mean to say that the linked article is a reference that covers the particulars of this rule in the criteria and therefore applies to all difficulties?

in that case, you may want to summarize the article's content into a few rule clauses and then link the new article for extra clarification

Reply:
it was because it was me typing a sentence at 1 am
the new sentence was not written at 1 am - the thing serves as clarification what something means and doesnt fit the overall style of the RC article so it's its separate thing